docs/02 - Contributing/F - Testing with AWS Services.md

```diff
@@ -0,0 +1,88 @@
+# Testing Anchor Platform with AWS Services
```

Comment:
💡  Nice doc file! Can you also add it to the table of content [here](https://github.com/stellar/java-stellar-anchor-sdk/tree/main/docs#table-of-content)?